informer/src/handler/connection.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{Events, Params, Sink, Value};
+use futures::Future;
+use jsonrpc_core::futures;
+
+#[derive(Clone)]
```

Comment:
When do you need to clone the Connection struct?

Reply:
I need to clone the connections of InfromerService when the code is notifying all clients.